tests/Unit/PointwiseFunctions/Hydro/CMakeLists.txt

```diff
@@ -19,5 +18,5 @@ add_test_library(
   ${LIBRARY}
   "PointwiseFunctions/Hydro/"
   "${LIBRARY_SOURCES}"
-  "Hydro;Test_GeneralRelativity;Test_Pypp"
+  "HydroHelpers;Test_GeneralRelativity"
```

Comment:
These really don't use pypp?

Reply:
pypp is now in `Framework`  which is linked by RunTests

Reply:
okay, we'll probably want to have individual libs depend on pypp, but that's for a future PR :)